docs/index.rst

```diff
@@ -30,7 +30,7 @@ Table of Contents
 .. toctree::
     :caption: Other Links
 
-    Download <https://github.com/adafruit/Adafruit_CircuitPython_Adafruit_IO/releases/latest>
+    Download <https://github.com/adafruit/Adafruit_CircuitPython_AdafruitIO/releases>
```

Comment:
Could you keep the `/latest` prefix and change the link to:
https://github.com/adafruit/Adafruit_CircuitPython_AdafruitIO/releases/latest